gpytorch/test/lazy_tensor_test_case.py

```diff
@@ -356,6 +356,14 @@ def test_add_diag(self):
                 actual[..., i, i] = actual[..., i, i] + other_diag[..., i]
             self.assertAllClose(res, actual, rtol=1e-2, atol=1e-5)
 
+    def test_cholesky(self):
```

Comment:
added this in b/c it/s never explicitly tested for most lazies (it is indirectly in `inv_quad_log_det` but this makes it explicit).